MEL/EwE/EwEShell/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@TeraNaidja so I setup to use packages and add Newtonsoft.Json for Ewe instead of direct add reference to a file (which was the wrong version...)